src/aat/resources/features/F-036/S-201.td.json

```diff
@@ -0,0 +1,44 @@
+{
+  "_guid_": "S-201",
+  "_extends_": "F-036_Base_Test_Data",
+  "title": "must return 401 when request does not provide valid authentication credentials",
+  "specs": [
+    "an active profile in CCD",
+    "does not provide valid authentication credentials in CCD",
+    "has a 403 Unauthorized code"
+  ],
+  "users": {
```

Comment:
"users" is already present in "F-036_Base_Test_Data"

Reply:
This scenario is where there is invalid credentials so I am overriding the password